src/coreComponents/fileIO/vtk/VTKPolyDataWriterInterface.hpp

```diff
@@ -210,7 +219,7 @@ class VTKPolyDataWriterInterface
   void clearData();
 
 
-private:
+protected:
```

Comment:
I may I missed something, but does anything inherits from `VTKPolyDataWriterInterface`?

Reply:
can be removed here because it is kept their: https://github.com/GEOS-DEV/GEOS/pull/2957/files#r1742425368